foreman-selinux-enable

```diff
@@ -13,15 +13,22 @@ trap "rm -rf '$TMP'" EXIT INT TERM
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    # UPGRADE STEP for Foreman 1.8 (remove after release)
+    # If katello policy was not yet loaded and elasticsearch port is still defined,
+    # remove it (it was moved to katello policy). Katello policy will not load before
+    # we remove the port definition (because SELinux prevents that).
+    semodule -l | grep -q katello || \
```

Comment:
What about not using elasticsearch_port_t in katello-selinux, so we remove all traces of it here, and then that package uses an entirely different name, like katello_allowed_ports_t.